Formula/appium.rb

```diff
@@ -18,7 +18,9 @@ class Appium < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    disable_chromedriver = ""
+    on_linux { disable_chromedriver << "--chromedriver-skip-install" }
```

Comment:
We probably want this regardless since IIRC it's prety much downloading an unversioned Chrome driver.

Reply:
That makes sense, and I think it's even possible for a user to download and install the Chrome driver as a separate package, so it's not a major inconvenience to disable everywhere.